docs/intro.rst

```diff
@@ -76,27 +78,31 @@ Important Notes/Caveats
 		* Multi-host clusters without using Swarm/Kubernetes host network is the best approach
 		* If you need clients to be able to access Kafka outside the bridge/overlay network
 
-4. Always launch containers with ``Restart=always`` unless you are using a process manager.
+4. Launch Settings
 
-5. These images are currently tested and shipped with `Azul Zulu OpenJDK <https://www.azul.com/products/zulu/>`_.  If you want to switch to Oracle Java, please refer to our instructions for extending the images
+    Docker containers should be launched with ``Restart=always`` unless you are using a process manager.   This ensures that intermittent failures in the Docker environment do not result in unnecessary failures of the Confluent services.
 
-6. Adding Connectors to the Kafka Connect Image
+5. Adding Connectors to the Kafka Connect Image
 
 	There are currently two ways to add new connectors to the Kafka Connect image.
 
-	* Build a new Docker image that has connector installed. You can follow the examples found in our documentation on `Extending Images <development.html#extending-the-docker-images>`_. You will need to make sure that the connector jars are on the classpath.
+	* Build a new Docker image that has the connector installed. You can follow the examples found in `Extending Images <development.html#extending-the-docker-images>`_. You will need to make sure that the connector jars are on the CLASSPATH for the Connect service (the default location of /usr/share/java/kafka-connect-* is the recommended location).
 	* Add the connector jars via volumes.  If you don't want to create a new Docker image, please see our documentation on `Configuring Kafka Connect with External Jars <operations/external-volumes.html>`_ to configure the `cp-kafka-connect` container with external jars.
 
+6. Included Java
+
+    The Confluent Docker images are tested and shipped with `Azul Zulu OpenJDK <https://www.azul.com/products/zulu/>`_.  Other JDK's (including Oracle Java) are supported, but you must extend the images yourself to implement that change. 
```

Comment:
are all other JDK options supported?  This makes it sounds like others are being tested.